sdk-setup/src/sdk-manage

```diff
@@ -681,14 +686,15 @@ register_object() {
     local object= domain=
     for object in "${objects[@]}"; do
         object=$type:$object
-        domain=$(enter "$object" sdk-register -d) || return
+        sdk_register=$(which_sdk_register "$object")
```

Comment:
hmm, only now I realized we could do even better by generalizing `which_sdk_register` as `object_which` and then

```suggestion
        sdk_register=$(object_which "$object" sdk-register)
```